### ISSUE TITLE
Support AWS Lambda URLs

### ISSUE THREAD

**Issue by diy-eng** [terraform-aws-modules/terraform-aws-lambda#306]
## Is your request related to a new offering from AWS?
Is this functionality available in the [AWS provider for Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)? See [CHANGELOG.md](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), too.
  - Yes ✅: please list the AWS provider version which introduced this functionality
  - [v4.9.0](https://github.com/hashicorp/terraform-provider-aws/blob/v4.9.0/CHANGELOG.md)
  - [Issue](https://github.com/hashicorp/terraform-provider-aws/issues/24050)

## Is your request related to a problem? Please describe.
The module does not support (AWS Lambda URLs)[https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html]

## Describe the solution you'd like.
The module will support AWS Lambda URLs.

## Describe alternatives you've considered.

Workaround: Use terraform native support.

**Comment by diy-eng** [comment]
Workaround
```
# LOCAL EXEC TO ENABLE URL
resource "null_resource" "create-function-url-config" {
  provisioner "local-exec" {

    command = "aws lambda create-function-url-config --function-name ${module.lambda_function.lambda_function_arn} --auth-type='NONE'"
  }

  depends_on = [
    module.lambda_function.webapp-api-backend
  ]
}
```

**Comment by antonbabenko** [comment]
This workaround is unnecessary because there is already native support for this resource in the Terraform AWS provider. We need to add it to the module, so PR is welcome.

**Comment by diy-eng** [comment]
> This workaround is unnecessary because there is already native support for this resource in the Terraform AWS provider. We need to add it to the module, so PR is welcome.

@antonbabenko Good call. I redacted the code & will try using using native support.

Will make some time to research if I can make PR, have not created/modified a module before.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.2.0 :tada:
